t/ui/12-needle-edit.t

```diff
@@ -465,7 +468,13 @@ subtest 'New needle instantly visible after reloading needle editor' => sub {
     $based_on_option->[0]->click();
     is($driver->find_element_by_id('tag_inst-timezone')->is_selected(),
         1, 'tag_inst-timezone checked again via new needle');
-
+    is($driver->find_element_by_id('property_workaround')->is_selected(),    1, 'workaround property selected');
+    is($driver->find_element_by_id('input_workaround_desc')->is_displayed(), 1, 'workaround description displayed');
+    is(
+        $driver->find_element_by_id('input_workaround_desc')->get_value(),
+        'bsc#123456 - this is a test',
+        'workaround description shows correctly'
```

Comment:
I would use `is shown`. It doesn't sound like s a common/valid use of ergative word order.